docs/examples/vector_stores/MongoDBAtlasVectorSearchRAGOpenAI.ipynb

```diff
@@ -0,0 +1,1478 @@
+{
```

Comment:
<p>pls remove logs to make it clean</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>